src/pages/additional-features/blueprint.mdx

```diff
@@ -0,0 +1,34 @@
+import { Callout } from 'nextra-theme-docs'
+
+# Blueprint
+
+Plutus Contract Blueprint, as introduced by [CIP-57](https://cips.cardano.org/cip/CIP-0057) is an effective way of communicating desired structure of validator's associated types such as datum & redeemer to your off-chain code.
+
+With Atlas, you can easily have these types generated for you based on provided schema, along with some helper utilities, yay!
+
+<Callout>
+
+If your validator is parameterised, parameters must be "data" encoded (instead of scott or sums-of-products encoding[^1]) as blueprint notation desires it (data) to be primary binary interface. This is not an issue for datums & redeemers as they are required to be data encoded anyway.
```

Comment:
The phrase "desires it" is a bit informal and could be clearer. Also, the sentence is a bit complex:

"parameters must be 'data' encoded (instead of using Scott or sums-of-products encoding[^1]) because blueprint notation requires data to be the primary binary interface."